pythoncalculator/subtract.py

```diff
@@ -0,0 +1,2 @@
+def subtract(x, y):
+    return y - y
```

Comment:
I think I've spotted a typo, perhaps try 
```suggestion
    return x - y
```